eva-accession-release/src/main/resources/create_assembly_name_symlinks.py

```diff
@@ -0,0 +1,56 @@
+import click
+import os
+import requests
+import glob
+import xml.etree.ElementTree
+from __init__ import *
+
+
+def get_assembly_name_for_accession(assembly_accession):
+    """
+    Get assembly name for a given assembly accession using ENA XML API.
+
+    Override value for GenScope12.X since it has a nonsensical name 12X from ENA:
+    https://github.com/EBIvariation/eva-accession/blob/56a8d6a5c42a7bf477c4623fcba21085c313bade/eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/AssemblyNameRetriever.java#L60
+
+    :param assembly_accession: GCA assembly accession
+    :return: Assembly name from ENA
+    """
+    if assembly_accession == "GCA_000003745.2":
+        return "Genoscope.12X"
+    ENA_XML_API_URL = "https://www.ebi.ac.uk/ena/browser/api/xml/" + assembly_accession
+    response = requests.get(ENA_XML_API_URL)
+    try:
+        if response.ok:
+            return xml.etree.ElementTree.fromstring(response.content.decode(encoding="utf-8"))\
+                .find("ASSEMBLY").find("NAME").text
+        else:
+            logger.error("API call {0} failed with response code {1}".format(ENA_XML_API_URL, response.status_code))
+    except Exception as ex:
+        logger.error(ex)
+
+
+def make_valid_filename_without_spaces(file_name):
+    return "".join([char for char in file_name if char.isalnum() or char in (' ', '.', '_')]).rstrip().replace(" ", "_")
+
+
+def add_assembly_name_symlink_for_dir(assembly_accession_release_dir):
+    parent_dir = os.path.dirname(assembly_accession_release_dir)
+    assembly_accession = os.path.basename(assembly_accession_release_dir)
+    target_dir_to_create = parent_dir + os.path.sep + \
+                           make_valid_filename_without_spaces(get_assembly_name_for_accession(assembly_accession))
+    logger.info('Creating symlink: "{0}" to {1}'.format(target_dir_to_create, assembly_accession_release_dir))
+    os.system('cd {0} && ln -sfT {1} "{2}"'.format(parent_dir, assembly_accession, target_dir_to_create))
```

Comment:
just to confirm, the syntax is `ln -sfT <link_destiny> <link_name>`, and it makes this (in the by_species/Mouse_10090/ folder):
```
GCA_000001635.4 -> ../../by_assembly/GCA_000001635.4
GCA_000001635.5 -> ../../by_assembly/GCA_000001635.5
GCA_000001635.6 -> ../../by_assembly/GCA_000001635.6
GRCm38.p2 -> GCA_000001635.4
GRCm38.p3 -> GCA_000001635.5
GRCm38.p4 -> GCA_000001635.6
mouse_10090_unmapped_ids.txt.gz
```
Looking at the code it was hard to understand, but this structure is ok